github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/events/impl/GitHubPROpenEvent/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger when PR is opened or commits (hash) changed in it.
+    Triggers a build of a pull request when the pull request is opened or reopened.
```

Comment:
Btw, reopen wouldn't work if trigger check will happen in some time moments? It triggers when previous check had no PR and now PR exists and has not `CLOSED` state. Probably it should also verify some time (note GH Issue and GH PR objects (api data) has different data).